tests/Test/Xrefcheck/AnchorsInHeadersSpec.hs

```diff
@@ -8,21 +8,21 @@ module Test.Xrefcheck.AnchorsInHeadersSpec where
 import Universum
 
 import Data.ByteString.Lazy qualified as BSL
-import Test.Hspec (Spec, describe, it, shouldBe)
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase, (@?=))
 
 import Xrefcheck.Core
 import Xrefcheck.Scanners.Markdown
 
-spec :: Spec
-spec = do
-    describe "Anchors in headers" $ do
-        it "Check if anchors in headers are recognized" $ do
+test_anchorsInHeaders :: TestTree
+test_anchorsInHeaders = testGroup "Anchors in headers" [
+          testCase "Check if anchors in headers are recognized" $ do
             fi <- getFI "tests/markdowns/without-annotations/anchors_in_headers.md"
-            getAnchors fi `shouldBe` ["some-stuff", "stuff-section"]
+            getAnchors fi @?= ["some-stuff", "stuff-section"]
 
-        it "Check if anchors with id attributes are recognized" $ do
+        ,  testCase "Check if anchors with id attributes are recognized" $ do
```

Comment:
This module was previously using 4-space indentation, even though we use 2-space everywhere else - can you please fix it while you're here?

Reply:
Done